outplacement_order_interpreter/__manifest__.py

```diff
@@ -22,7 +22,8 @@
 # Version format OdooMajor.OdooMinor.Major.Minor.Patch
 {
     'name': 'Order Interpreter',
-    'version': '12.0.1.7.8',
+    'version': '12.0.1.7.3',
```

Comment:
@nils-vertel Here in this manifest file two lines for version added. There must be only one.

Reply:
Great catch, bad merge, fixed now.
Any other problems with the code?

Reply:
No @nils-vertel